ts/webui/src/components/overview/count/EditExperimentParam.tsx

```diff
@@ -63,20 +64,32 @@ export const EditExperimentParam = (): any => {
         }
     }
 
+    function promptErrorMessage(mess: string, type: string, value: string): void {
+        showMessageInfo(mess, type);
+        setEditValInput(value);
+    }
+
     async function confirmEdit(): Promise<void> {
         const isMaxDuration = title === 'Max duration';
         const newProfile = Object.assign({}, EXPERIMENT.profile);
         let beforeParam = '';
         if (isMaxDuration) {
             if (!editInputVal.match(/^\d+(?=\.{0,1}\d+$|$)/)) {
-                showMessageInfo('Please enter a number!', 'error');
-                setEditValInput(defaultVal);
+                promptErrorMessage('Please enter a number!', 'error', defaultVal);
+                return;
+            }
+            if (toSeconds(`${editInputVal}${unit}`) < EXPERIMENT.profile.execDuration) {
+                // maxDuration should > current run time(execDuration)
+                promptErrorMessage(
+                    'Current duration is more than the number you input, please input effective value',
```

Comment:
please input a valid value. (Current duration is ...)

Reply:
I had changed it to `Please input a valid value. (Current duration is more than the number you input.)`. Please help review again~